.githooks/pre-commit

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# Make sure /usr/local/bin is in the path
+PATH="$PATH:/usr/local/bin"
+
+# Check if there are any staged files ending with `.ts`
+STAGED_FILES=$(git diff --cached --name-only --diff-filter=ACM | grep "\.ts$")
+
+if [[ "$STAGED_FILES" = "" ]]; then
+  exit 0
+fi
```

Comment:
This is way too specific for git hook - separating concerns. Hook should just run some commands at different stages, not care about what the command will do. 

As an example, once there are other types of files to lint, we'll surely update the lint command, but this will likely be forgotten meaning the lint won't run if there are no TS files in the commit. 
IMO we should just let lint command run always.

Reply:
Sounds reasonable, let's rip it out again.